sklearn/sklearn-template/template/trainer/model.py

```diff
@@ -20,34 +20,48 @@
 from sklearn import pipeline
 from sklearn import preprocessing
 
+import numpy as np
+
 from trainer import metadata
 
 
 def get_estimator(flags):
-  classifier = ensemble.RandomForestClassifier()
-
-  # TODO(cezequiel): Make use of flags for hparams
-  _ = flags
+  # TODO: Allow pre-processing to be configurable through flags
+  classifier = ensemble.RandomForestClassifier(**flags)
```

Comment:
Take note that `flags` is actually the object returned by `ArgumentParser.parse_args` might be better to 
1. Specify the parameters explicitly, e.g. `flags.n_estimators` in the classifier declaration.
2. Filter `flags` upstream so you get just the ones needed by this function (you might want to rename `flags` to `hparams` or something similar in that case, and then possibly add a `model_type` parameter to `get_estimator`)

Reply:
I modify it according to your first suggestion. I think this is simpler and align with the purpose of template.